README.md

```diff
@@ -0,0 +1,94 @@
+# gr-stellarstation: GNURadio blocks for interacting with StellarStation
+
+gr-stellarstation is an out-of-tree GNURadio module that provides blocks for easily integrating with the [StellarStation](https://www.stellarstation.com/) API in GNURadio flowgraphs.
+
+## Installation
+
+### PyBOMBS
+
+PyBOMBs support is currently under construction.
+
+### Source Installation
+
+gr-stellarstation, like most OOT modules, can be built from source using CMake.
+
+First, we need to install gr-stellarstation's sole dependency, [Conan](https://conan.io/). The simplest way to do this is through `pip`:
+
+```bash
+$ pip install conan
+```
+
+For alternative methods to install Conan, see the [Conan documentation](https://docs.conan.io/en/latest/installation.html).
+
+After installing Conan, run the following commands to add the relevant Conan recipes:
+
+```bash
+$ conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
+$ conan remote add inexorgame https://api.bintray.com/conan/inexorgame/inexor-conan
+$ conan remote add stellarstation https://api.bintray.com/conan/infostellarinc/stellarstation-conan
+```
+
+Proceed to install gr-stellarstation normally with CMake:
+
+```bash
+$ git clone https://github.com/infostellarinc/gr-stellarstation.git
+$ cd gr-stellarstation
+$ mkdir build
+$ cmake ..
+$ make
+$ sudo make install
```

Comment:
I prefer not including `sudo` in installation command since it is a somewhat risky operation to install something new to their system which might cause damage, and I want to prevent accidental installation by those who don't have much knowledge about what they are doing.  Rather than writing it as a instruction, we can leave it to users by just writing commands without `sudo` with a comment saying "You will need  superuser privilege to execute installation commands."
```
$ sudo make install
$ sudo ldconfig
```

Reply:
Makes sense.